Marlin/src/HAL/TEENSY35_36/HAL.h

```diff
@@ -54,8 +54,12 @@
 #endif
 
 #include "../../core/serial_hook.h"
-typedef Serial1Class<decltype(Serial)> DefaultSerial1;
-extern DefaultSerial1 MSerial0;
+
+#define _DECLARE_SERIAL(X) \
+  typedef Serial1Class<decltype(Serial##X)> DefaultSerial##X; \
```

Comment:
You need the `#define Serial0 Serial` for this line. Or just drop this commit, since there are 3 similar versions of Teensy to fix this way, I can do it in another PR.

Reply:
nevermind.